net/udp-broadcast-relay-redux-openwrt/Makefile

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2006-2014 OpenWrt.org
```

Comment:
Please use your own Copyright, or leave it blank.

Reply:
Fixed.  Also removed from the init script.